python/cudf/tests/test_string.py

```diff
@@ -311,7 +311,27 @@ def test_string_len(ps_gs):
 @pytest.mark.parametrize('others', [
     None,
     ['f', 'g', 'h', 'i', 'j'],
-    pd.Series(['f', 'g', 'h', 'i', 'j'])
+    ('f', 'g', 'h', 'i', 'j'),
+    pd.Series(['f', 'g', 'h', 'i', 'j']),
+    (['f', 'g', 'h', 'i', 'j'], ['f', 'g', 'h', 'i', 'j']),
+    [['f', 'g', 'h', 'i', 'j'], ['f', 'g', 'h', 'i', 'j']],
+    (pd.Series(['f', 'g', 'h', 'i', 'j']),
+     pd.Series(['f', 'g', 'h', 'i', 'j'])),
+    [pd.Series(['f', 'g', 'h', 'i', 'j']), pd.Series(
+        ['f', 'g', 'h', 'i', 'j'])] * 20,
+    (
+        pd.Series(['f', 'g', 'h', 'i', 'j']),
+        pd.Series(['f', 'g', 'h', 'i', 'j']),
+        pd.Series(['f', 'g', 'h', 'i', 'j']),
+        pd.Series(['f', 'g', 'h', 'i', 'j']),
+        ['f', 'a', 'b', 'f', 'a']
+    ),
+    [
+        ['f', 'a', 'b', 'f', 'a'], pd.Series(['f', 'g', 'h', 'i', 'j']),
+        pd.Series(['f', 'g', 'h', 'i', 'j']),
+        pd.Series(['f', 'g', 'h', 'i', 'j']),
+        pd.Series(['f', 'g', 'h', 'i', 'j'])
+    ]
```

Comment:
Should add a case for `pd.Index` as well.

Reply:
Added more test combinations